trySwift/SpeakerTableViewCell.swift

```diff
@@ -26,6 +26,7 @@ class SpeakerTableViewCell: UITableViewCell {
 
     override func awakeFromNib() {
         super.awakeFromNib()
+        speakerTwitterButton.titleLabel?.adjustsFontForContentSizeCategory = true
```

Comment:
UI buttons do not have the "Dynamic Type" check box in the UI editor; thus we must programmatically enable dynamic text.